docs/jupyterlite.rst

```diff
@@ -0,0 +1,25 @@
+Try out now with JupyterLite_
+-----------------------------
+
+.. admonition:: To try pyhf code in the browser with the Pyolite JupyterLite kernel:
+    :class: dropdown
+
+     #. Type (or copy and paste) code in the input cell.
+     #. To execute the code, press ``Shift + Enter`` or click on the (Run) ▶ button in the toolbar.
+
+     To get going try copying and pasting the "Hello World" example below!
+
```

Comment:
@kratsg @alexander-held Do you think that this info would be helpful? Or is this overestimating the problem and severity?

```suggestion

     .. admonition:: Kernel not loading?
      :class: dropdown

       If the kernel isn't loading and you just see a blank iframe, you will need your
       browser to accept cookies from jupyterlite.github.io and then reload the page.

```

Reply:
Ah I see Alex already said this would be helpful. I'll add.

Reply:
Definitely feel free to not include it, I just figured it does not hurt to have and may help a few people.